WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -1,7 +1,7 @@
 <ul class="list-group">
   <li class="list-group-item"><strong><%= title %></strong></li>
   <% competitions.each_with_index do |competition, index| %>
-    <% if index > 0 && competition.year != competitions[index - 1].year %>
+    <% if index > 0 && competition.year != competitions[index - 1].year && params[:event_ids].empty? %>
```

Comment:
Purpose: assume we have October right now, thus having a lot of upcoming comps in both 2016 and 2017. Sorting by matched events would mix the list, so we don't want to have a lot of labels 2016 or 2017.